gslib/commands/cp.py

```diff
@@ -679,6 +679,10 @@
                  CAUTION: If some of the source files don't compress well, such
                  as binary data, using this option may result in files taking up
                  more space in the cloud than they would if left uncompressed.
+
+  --stet         If the STET binary can be found in boto or PATH, and STET
+                 config file can be found in boto or default location, cp will
```

Comment:
The criteria listed here don't seem quite right. If the STET config isn't in the boto, gsutil shouldn't be trying to find any default location; it should call stet without passing any config file (letting stet deal with finding a default config file).

Reply:
Fixed